### PR TITLE
feat(website): Partner announcement

### PR DESCRIPTION
**Pull request by jsonpr**
AWS Partner Announcement Blog

### CODE REVIEW

File: website/pages/blog/announcing-aws-partnership.mdx [hunk 1]
```diff
@@ -0,0 +1,74 @@
+---
+title: Announcing CloudQuery as an Official AWS Partner
+tag: announcement
+date: 2023/08/24
+description: >-
+  CloudQuery has been recognized as an Amazon Web Services Partner and validated via a foundational technical review.  This Partner program will enable CloudQuery to build and service more customer needs including security, cost optimization, infrastructure, and development.
```
Comment (candiduslynx) [inline website/pages/blog/announcing-aws-partnership.mdx:6]:
```suggestion
  CloudQuery has been recognized as an Amazon Web Services Partner and validated via a foundational technical review. This Partner program will enable CloudQuery to build and service more customer needs including security, cost optimization, infrastructure, and development.
```

File: website/pages/blog/announcing-aws-partnership.mdx [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+author: jsonkao
+---
+
+import { BlogHeader } from "../../components/BlogHeader"
+
+<BlogHeader/>
+
+We are excited to announce that CloudQuery has officially been recognized as a [member of the AWS Partner Network](https://partners.amazonaws.com/partners/0010h00001kNW07AAG).  From this partnership program, we're looking forward to continued development on extracting, transforming, and loading data from AWS APIs to a variety of supported destinations that include databases, data lakes, and streaming platforms for further analysis.  CloudQuery supports multiple use cases built on top of this data platform including security, cost optimization, infrastructure, and development.
```
Comment (candiduslynx) [inline website/pages/blog/announcing-aws-partnership.mdx:14]:
```suggestion
We are excited to announce that CloudQuery has officially been recognized as a [member of the AWS Partner Network](https://partners.amazonaws.com/partners/0010h00001kNW07AAG). From this partnership program, we're looking forward to continued development on extracting, transforming, and loading data from AWS APIs to a variety of supported destinations that include databases, data lakes, and streaming platforms for further analysis. CloudQuery supports multiple use cases built on top of this data platform including security, cost optimization, infrastructure, and development.
```

File: website/pages/blog/announcing-aws-partnership.mdx [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+![AWS Partner Badge](/images/blog/announcing-aws-partnership/partnerbadge.png)
+
+We have a exiting roadmap ahead of new features and improvements.  Stay tuned!
```
Comment (candiduslynx) [inline website/pages/blog/announcing-aws-partnership.mdx:18]:
```suggestion
We have a exiting roadmap ahead of new features and improvements. Stay tuned!
```

File: website/pages/blog/announcing-aws-partnership.mdx [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+## Highlights
+
+* CloudQuery was recently featured on an AWS live streamed data show, Let's Talk About Data: https://www.twitch.tv/videos/1894215221
+
+* Part of the partner process involved us working with AWS to complete multiple steps including an AWS Foundational Technical Review to ensure our offerings meet AWS's stringent technical requirements and provide value to customers. 
+
+## How CloudQuery works with Amazon Web Services
+
+CloudQuery supports a variety of [source plugins](https://www.cloudquery.io/docs/plugins/sources/overview) and [destination plugins](https://www.cloudquery.io/docs/plugins/destinations/overview).  By using these source plugins and destinations, information from AWS configurations and deployments can be synced.  Additionally, infrastructure data from a variety of sources can be synced to AWS-related and hosted data destinations.
```
Comment (candiduslynx) [inline website/pages/blog/announcing-aws-partnership.mdx:28]:
```suggestion
CloudQuery supports a variety of [source plugins](https://www.cloudquery.io/docs/plugins/sources/overview) and [destination plugins](https://www.cloudquery.io/docs/plugins/destinations/overview). By using these source plugins and destinations, information from AWS configurations and deployments can be synced. Additionally, infrastructure data from a variety of sources can be synced to AWS-related and hosted data destinations.
```

File: website/pages/blog/announcing-aws-partnership.mdx [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+* Unused Resources in AWS
+* Kubernetes 
+
+Please see our links for policies and dashboards for more information about these offerings:
+* [Policies](https://www.cloudquery.io/docs/core-concepts/policies)
+* [Dashboards](https://www.cloudquery.io/docs/core-concepts/dashboards)
+
+## Use Cases
+
+CloudQuery supports multiple use cases including security, cost optimization, infrastructure, and development.  Below is a sampling of the use cases that can be built with CloudQuery with AWS infrastructure data:
```
Comment (candiduslynx) [inline website/pages/blog/announcing-aws-partnership.mdx:64]:
```suggestion
CloudQuery supports multiple use cases including security, cost optimization, infrastructure, and development. Below is a sampling of the use cases that can be built with CloudQuery with AWS infrastructure data:
```

File: website/pages/blog/announcing-aws-partnership.mdx [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+
+* [Open Source Cloud Security Posture Management (CSPM)](https://www.cloudquery.io/how-to-guides/open-source-cspm)
+* [Open Source Attack Surface Management with Neo4j (ASM)](https://www.cloudquery.io/how-to-guides/attack-surface-management-with-graph)
+* [Analyzing AWS Cost and Usage Data](https://www.cloudquery.io/how-to-guides/query-aws-cost-and-usage-athena)
+* [Managing AWS Resource and Usage Limits](https://www.cloudquery.io/how-to-guides/aws-resource-limits-and-service-quotas)
+* [Managing Resilience Strategy with AWS Backup](https://www.cloudquery.io/how-to-guides/manage-resilience-with-aws-backup)
+
+## Get Started
+
+We'd love to hear your thoughts.  We can be reached on [GitHub](https://github.com/cloudquery/cloudquery/issues), [Discord](https://www.cloudquery.io/discord).
```
Comment (candiduslynx) [inline website/pages/blog/announcing-aws-partnership.mdx:74]:
```suggestion
We'd love to hear your thoughts. We can be reached on [GitHub](https://github.com/cloudquery/cloudquery/issues), [Discord](https://www.cloudquery.io/discord).
```

File: website/pages/blog/announcing-aws-partnership.mdx (hunk 3 above)
Comment (hermanschaaf) [inline website/pages/blog/announcing-aws-partnership.mdx:18]:
```suggestion
We have an exciting roadmap ahead of new features and improvements.  Stay tuned!
```

Comment (jsonpr) [reply website/pages/blog/announcing-aws-partnership.mdx]: Thanks!  Fixed.

**Review by yevgenypats — APPROVED** [review]

**Pull request merged**